NewsApp/NewsApp/.swiftlint.yml

```diff
@@ -0,0 +1,45 @@
+disabled_rules:
+
+trailing_whitespace:
+nesting:
+variable_name:
+large_tuple:
+multiple_closures_with_trailing_closure:
```

Comment:
Sai format rồi em ơi
disabled_rules: # rule identifiers turned on by default to exclude from running
  - trailing_whitespace
  - nesting
...